action.yml

```diff
@@ -6,89 +6,29 @@ branding:
   icon: 'package'
 
 inputs:
+  project:
+    description: 'Required: The name or ID of the project associated with this release.'
```

Comment:
Instead of putting `Required:` in the description (which won't enforce it being set) you can use `required: true` as a property on the `project` input (which will enforce is being set) https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired

Reply:
Fixed in later commit to this branch. Thanks!